src/Interactive/EditorFeatures/Core/Extensibility/Interactive/ResetInteractive.cs

```diff
@@ -104,11 +104,12 @@ internal Task Execute(IInteractiveWindow interactiveWindow, string title)
                 await interactiveEvaluator.SetPathsAsync(referenceSearchPaths, sourceSearchPaths, projectDirectory).ConfigureAwait(true);
             }
 
-            await interactiveWindow.SubmitAsync(new[]
+            var importReferencesCommand = referencePaths.Select(_createReference);
+            var importNamespacesCommand = namespacesToImport.Select(_createImport).Join("\r\n");
```

Comment:
Use `Environment.NewLine` (or whatever shim we have in place to accomplish the same).